debian/changelog

```diff
@@ -1,3 +1,27 @@
+opx-nas-linux (5.25.0) unstable; urgency=medium
+
+  * Feature: VxLAN Support
+  * Bugfix: Fixed issue of dst_ip not set to system
```

Comment:
what is dst_ip? Please write the real word for it (Destination IP?)
And grammar is little broken here. More context would be better.

Reply:
This should say `* Bugfix: dst_ip not being filtered for in CoPP ACL for mgmt VRF`